### DISCUSSION TITLE
Tailpress and plesk

### DISCUSSION THREAD

**Discussion by gdavey** [tailpress/tailpress#91] [Q&A] [answered]
Hi,

I run a gaming community https://merric-gaming.co.uk/ and I wanted to use tailpress to replace our current theme, I was wondering if anyone has had any experience getting this working on plesk? 

I have tried.
`/opt/plesk/php/7.4/bin/php /usr/lib64/plesk-9.0/composer.phar global require jeffreyvanrossum/tailpress-installer`

When I try to run 
`tailpress new example-theme`

I get an output of `tailpress: command not found`

Any help would be appreciated.

**Comment by jeffreyvr** [comment]
Hi @gdavey Personally I would develop your theme locally and then, once you're finished, upload it to your WordPress instance. That way you won't have to use composer, node, npm etc. on your Plesk server.

I'm not sure how you can make Plesk recognise the command, but you could instead run the command from the composer vendor/bin folder (For example `.composer/vendor/bin/tailpress`).

**Comment by gdavey** [comment]
Hi @jeffreyvr 

I seem to be getting somewhere, by cloning the repo and run `/opt/plesk/node/16/bin/npm install`

`473 error path FULL SERVER PATH/tailpress/node_modules/esbuild`
`474 error command failed`
`475 error command sh -c node install.js`
`476 error sh: node: command not found`
`477 verbose exit 127`

**Comment by gdavey** [comment, marked as answer]
Managed to get this working.

For future reference anyone using Plesk do the following.

`export PATH=/opt/plesk/node/16/bin:$PATH`
`echo "export PATH=/opt/plesk/node/16/bin:$PATH" >> ~/.bashrc`

`npm install`
